src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -1867,6 +1867,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageService
             initializer = objectCreationExpression.Initializer
         End Sub
 
+        Public Sub GetPartsOfImplicitObjectCreationExpression(node As SyntaxNode, ByRef keyword As SyntaxToken, ByRef argumentList As SyntaxNode, ByRef initializer As SyntaxNode) Implements ISyntaxFacts.GetPartsOfImplicitObjectCreationExpression
+            Throw New InvalidOperationException(DoesNotExistInVBErrorMessage)
```

Comment:
nit add comment that vb doesn't have this construct.

Reply:
This pattern is used in many locations in this file. It seems like the identifier `DoesNotExistInVBErrorMessage` makes a comment redundant.

Reply:
ah, i missed the message. this is fine with me.